File: Estrutura_Condicional/Java/Ex1045/Main.java
package Ex1045;

import java.util.Arrays;
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        double a = scanner.nextDouble();
        double b = scanner.nextDouble();
        double c = scanner.nextDouble();
        scanner.close();

        double[] lados = {a, b, c};
        Arrays.sort(lados);

        a = lados[2];
        b = lados[1];
        c = lados[0];

        if (a >= b + c) {
            System.out.println("NAO FORMA TRIANGULO");
        } else {
            if (Math.pow(a, 2) == Math.pow(b, 2) + Math.pow(c, 2)) {
                System.out.println("TRIANGULO RETANGULO");
            }

            if (Math.pow(a, 2) > Math.pow(b, 2) + Math.pow(c, 2)) {
                System.out.println("TRIANGULO OBTUSANGULO");
            }

            if (Math.pow(a, 2) < Math.pow(b, 2) + Math.pow(c, 2)) {
                System.out.println("TRIANGULO ACUTANGULO");
            }

            if (a == b && b == c) {
                System.out.println("TRIANGULO EQUILATERO");
            }

            if (a == b && b != c || a == c && c != b || b == c && c != a) {
                System.out.println("TRIANGULO ISOSCELES");
            }
        }
    }
}

File: Estrutura_Condicional/C#/Ex1045/Main.cs
using System;

namespace Ex1045
{
    class Program
    {
        static void Main(string[] args)
        {
            double a, b, c;
            string[] inputs = Console.ReadLine().Split(' ');
            a = double.Parse(inputs[0]);
            b = double.Parse(inputs[1]);
            c = double.Parse(inputs[2]);

            double[] lados = { a, b, c };
            Array.Sort(lados);

            a = lados[2];
            b = lados[1];
            c = lados[0];

            if (a >= b + c)
            {
                Console.WriteLine("NAO FORMA TRIANGULO");
            }
            else
            {
                if (Math.Pow(a, 2) == Math.Pow(b, 2) + Math.Pow(c, 2))
                {
                    Console.WriteLine("TRIANGULO RETANGULO");
                }

                if (Math.Pow(a, 2) > Math.Pow(b, 2) + Math.Pow(c, 2))
                {
                    Console.WriteLine("TRIANGULO OBTUSANGULO");
                }

                if (Math.Pow(a, 2) < Math.Pow(b, 2) + Math.Pow(c, 2))
                {
                    Console.WriteLine("TRIANGULO ACUTANGULO");
                }

                if (a == b && b == c)
                {
                    Console.WriteLine("TRIANGULO EQUILATERO");
                }

                if (a == b && b != c || a == c && c != b || b == c && c != a)
                {
                    Console.WriteLine("TRIANGULO ISOSCELES");
                }
            }
        }
    }
}